packages/react-native/V6.js

```diff
@@ -0,0 +1 @@
+export * from './dist/V6';
```

Comment:
question: why is this file necessary?

Reply:
Its for backwards compatibility with the v6 store. In v7 there is a new story store which changes the way that we load stories and simplifies the setup.

Reply:
For example storiesOf requires the v6 store and isn't compatible with the v7 store